src/aqua/graphics/lat_lon_profiles.py

```diff
@@ -0,0 +1,75 @@
+import xarray as xr
+import matplotlib.pyplot as plt
+
+from aqua.logger import log_configure
+from .util_timeseries import plot_monthly_data, plot_annual_data
+from .styles import ConfigStyle
+
+def plot_lat_lon_profiles(mean_type: str, 
+                          monthly_data: xr.DataArray = None,
+                          annual_data: xr.DataArray = None,
+                          data_list: list = None,
+                          data_labels: list = None,
+                          style: str = None,
+                          fig: plt.Figure = None, 
+                          ax: plt.Axes = None,
+                          loglevel='WARNING',
+                          **kwargs):
+
+
+    logger = log_configure(loglevel, 'plot_lat_lon_profiles')
+    ConfigStyle(style=style, loglevel=loglevel)
+
+    def data_coordinate_means(data, mean_type : str):
+        """
+        Calculate the mean of the data along the latitude or longitude axis.
+        """
+        logger.debug(f"{mean_type} mean calculation")
+        if mean_type == 'zonal':
+            return data.mean(dim='lon')
+        elif mean_type == 'meridional':
+            return data.mean(dim='lat')
+        else:
+            raise ValueError("mean_type must be 'zonal' or 'meridional'")
+        
+
+    if data_list is not None:
```

Comment:
If data_list is not none what is plotted monthly data or annual_data?  I would pass only one "data" object. Then you can test if it a slist (and iterate over it) or do only one plot